cpp/libcudf_streaming/benchmarks/streaming/bench_streaming_shuffle.cpp

```
@@ -0,0 +1,384 @@
+    log.print(ss.str());
+    if (i >= args.num_warmups) { elapsed_vec.push_back(elapsed); }
+  }
+
+  if (!use_bootstrap) {
+    RAPIDSMPF_MPI(MPI_Barrier(MPI_COMM_WORLD));
+  } else {
+    std::dynamic_pointer_cast<rapidsmpf::ucxx::UCXX>(comm)->barrier();
+  }
+
+  {
+    auto const elapsed_mean = harmonic_mean(elapsed_vec);
```

Cppcheck (2.20.0)

[error] Unhandled exception thrown in function that is an entry point.

(throwInEntryPoint)